docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+#Coordinator Model
+
+## Table of Contents
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+1. [Role of the Coordinator](#role-of-a-coordinator)
+1. [Change Management](#change-management)
+1. [Issue Management](#issue-management)
+1. [Scenarios](#scenarios)
+1. [What can be expected of a coordinator?](#expectations)
+
+
+<!-- /MarkdownTOC -->
+<a name="role-of-a-coordinator"></a>
+## Role of the Coordinator
+
+The Coordinator is not the owner of the component but much more the curator. 
+The coordinator ensures the quality of contributions to the component 
+and makes it possible for others to commit. The coordinator is responsible 
+that the documentation is kept up to date by the contributors and that the 
+guidelines of the component are met. The coordinator moderates the discussion on 
+finding a vision and on the development. The coordinator further is contact 
+for any sort of question that may arise about the respective component.
+
+The motivation of the coordinator is mostly driven by the need of a 
+reliable component for a certain aspect. Further the coordinator is 
+probably the most attractive contractor for clients aiming to change 
+aspects of the component due to the very indepth know how and the 
+listing as coordinator.
+
+<a name="change-management"></a>
+##Change Management
+Everybody may contribute to any aspect of the component. Such contributions 
+are handed in by pull requests or some other source of data if declared so in the components guidelines. 
+Pull requests on the public interface must be accepted by the JF. The coordinator gives a 
+recommendation to the JF on whether to accept or decline the PR. The decision of the JF may 
```

Comment:
`The decision of the JF may be implicit if no objections to the recommendation of the coordinator is made.` seem a bit unclear to me: A coordinator gives a recommondation and appends the JF-Label, how long should he wait (e.g. if the PR isn't discussed during the next JF?). Maybe this is just a very minor issue

Reply:
True, however, this is the best we have been able to come up with. maybe we can improve that after gathering some experience.